core/src/main/java/org/elasticsearch/cluster/health/ClusterHealthStatus.java

```diff
@@ -38,7 +44,24 @@ public byte value() {
         return value;
     }
 
-    public static ClusterHealthStatus fromValue(byte value) {
+    @Override
+    public void writeTo(StreamOutput out) throws IOException {
+        out.writeByte(value);
+    }
+
+    /**
+     * Load a {@link ClusterHealthStatus}.
```

Comment:
I usually just do
```
/**
 * Read from a stream.
 */
```

The input and output are relatively obvious. Same for the IOException. The IllegalArgumentException isn't, but honestly it isn't a big deal I think.

Also not a big deal, more a matter of my personal style preference which I don't want to force on anyone.